Lib/test/eintrdata/eintr_tester.py

```diff
@@ -507,7 +507,7 @@ def _lock(self, lock_func, lock_name):
                         lock_func(f, fcntl.LOCK_EX | fcntl.LOCK_NB)
                         lock_func(f, fcntl.LOCK_UN)
                         time.sleep(0.01)
-                    except BlockingIOError:
+                    except (BlockingIOError, PermissionError):
```

Comment:
This and the related NEWS entry should be removed from the PR, as per the discussion in https://bugs.python.org/issue35633 (I'm guessing it's an artifact of this PR being based the initial version of that one).

Reply:
done.